docs/source/xformers_mingpt.ipynb

```diff
@@ -7,7 +7,7 @@
         "id": "view-in-github"
       },
       "source": [
-        "<a href=\"https://github.com/facebookresearch/xformers/blob/colab_example/docs/source/xformers_mingpt.ipynb\" target=\"_parent\"><img src=\"https://colab.research.google.com/assets/colab-badge.svg\" alt=\"Open In Colab\"/></a>"
```

Comment:
I don't have much context on this, so I'm gonna ask some stupid questions. The old link was referring to a file in the `colab_example` branch, but that branch doesn't seem to exist anymore. The file in question, however, seems to exist in the `main` branch, under the same path. Is that the same file?

If so, why not just fix the name of the branch?

What do we gain from linking to the colab website?

Reply:
You are correct, from the the branch name I assumed it should point to Colab, but I changed it to `main`.